python/trezorlib/tests/device_tests/test_msg_recoverydevice_shamir_dryrun.py

```diff
@@ -0,0 +1,78 @@
+import time
+
+import pytest
+
+from trezorlib import debuglink, device, messages as proto
```

Comment:
please never use `messages as proto`, just rename "proto" to "messages"

(feel free to do that if you're editing old files that use this idiot)

Reply:
done